ridgeback_dual_gen3_moveit_config/launch/trajectory_execution.launch.xml

```diff
@@ -7,9 +7,9 @@
   <param name="moveit_manage_controllers" value="$(arg moveit_manage_controllers)"/>
 
   <!-- When determining the expected duration of a trajectory, this multiplicative factor is applied to get the allowed duration of execution -->
-  <param name="trajectory_execution/allowed_execution_duration_scaling" value="1.2"/> <!-- default 1.2 -->
+  <param name="trajectory_execution/allowed_execution_duration_scaling" value="2.0"/> <!-- default 1.2 -->
   <!-- Allow more than the expected execution time before triggering a trajectory cancel (applied after scaling) -->
-  <param name="trajectory_execution/allowed_goal_duration_margin" value="0.5"/> <!-- default 0.5 -->
+  <param name="trajectory_execution/allowed_goal_duration_margin" value="2.0"/> <!-- default was 0.5 but it needs to be higher because preprocessing in the arm takes time -->
```

Comment:
I believe there is a parameter to disable the trajectory execution timeout. Did you look into that option at all? I'm fine with increasing the margin if that other option either doesn't exist or wasn't reliable.

Reply:
I didn't even consider is because I just copied what Kinova does in their driver. I tested this and it works reliably, so I don't see a reason to disable it. And, would disabling a timeout cause issues in the even that the Kinova driver cannot execute a trajectory but MoveIt! is waiting for a response?

Reply:
I also just added the dual arm tower mount. It doesn't really belong in the ridgeback_description package since it's an arm mount like the FAMS and HAMS. Is there another name that would be better than `dual_arm_tower`?

Reply:
I like `dual_arm_tower`. It's descriptive and works. And given its size we're unlikely to run into naming conflicts with a different `dual_arm_tower` present on the same robot.